lgc/util/GfxRegHandler.cpp

```diff
@@ -246,8 +247,7 @@ Value *SqImgRsrcRegHandler::getReg(SqRsrcRegs regId) {
       llvm_unreachable("GFX IP is not supported!");
       break;
     }
-  case SqRsrcRegs::IsTileOpt:
-    return m_builder->CreateICmpNE(getRegCommon(static_cast<unsigned>(regId)), m_builder->getInt32(0));
+    return getRegCommon(static_cast<unsigned>(regId));
```

Comment:
This is superfluous, as all cases are handled in the switch statement.

Reply:
> This is superfluous, as all cases are handled in the switch statement.

Oh, this should be deleted.